test-lib-openshift.sh

```diff
@@ -344,6 +345,89 @@ function ct_os_cluster_running() {
   oc cluster status &>/dev/null
 }
 
+# ct_os_set_path_oc OC_VERSION
+# --------------------
+# This is a trick that helps using correct version of the `oc`:
+# The input is version of the openshift in format v3.6.0 etc.
+# If the currently available version of oc is not of this version,
+# it first takes a look into /usr/local/oc-<ver>/bin directory,
+# and if not found there it downloads the community release from github.
+# In the end the PATH variable is changed, so the other tests can still use just 'oc'.
+# Arguments: oc_ver - X.Y part of the version of OSE (e.g. 3.9)
```

Comment:
s/oc_ver/OC_VERSION/

Reply:
I used deliberately `oc_ver`, since that is the local variable that the value is stored in the function, and it's done similarly in other functions. Although I'm not familiar with shell conventions, so maybe we can sync it in all doctexts in the file.

Reply:
Ok, then s/OC_VERSION/oc_ver/ in header?  It's just inconsistency in function docs.

Reply:
Other images in this file use same convention for parameters. Capitals above `# ---------------` and small letters in description.

So we should then fix all descriptions.

Reply:
Well, whether that's oc_ver or OC_VER doesn't matter; more important is `oc_ver != oc_version`.

Reply:
ah, right, will use full name oc_version then.